File: Cpp/CS101-Programming-Fundamentals/Assignments/Assignment 2/Task 1.cpp
#include <iostream>
using namespace std;
int main(){
	int num, remainder, reverse;
	cout<<"Enter a number : ";
	cin>>num;
	
	while(num !=0){
		remainder = num % 10;
		reverse = reverse * 10 + remainder;
		num = num / 10;
		
	}
	cout <<"Reversed number is: "<<reverse ;
	
	return 0;
	}

File: Python/AI-102-Python-and-Freelancing-Essencials/Lab-Tasks/Lab 4/Task 1.py
print()
print("Finding Even or Odd")
 
def func(n):            #defining the function to find even or odd
    if n%2==0:          #if the remainder with 2 is 0 then it is even
        print("The number is even.")
    else :              #if the remainder with 2 is not 0 then it is odd
        print("The number is odd.")
    
while True:             #exception handling
    num = int(input("Enter a number(-100 to exit): "))
    if num == -100:     #condition for exiting the program
        print("Program Exited")
        break
    elif num == 0 :
        print("0 is neither odd nor even")
    else :
        func(num)       #calling the function